app/routes/events/view/speakers/list.js

```diff
@@ -1,15 +1,13 @@
 import Route from '@ember/routing/route';
 import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
+import { capitalize } from 'lodash-es';
 
 export default class extends Route.extend(EmberTableRouteMixin) {
   titleToken() {
-    switch (this.params.speakers_status) {
-      case 'pending':
-        return this.l10n.t('Pending');
-      case 'accepted':
-        return this.l10n.t('Accepted');
-      case 'rejected':
-        return this.l10n.t('Rejected');
+    if (['accepted', 'rejected', 'confirmed', 'withdrawn', 'canceled'].includes(this.params.speakers_status)) {
+      return this.l10n.tVar(capitalize(this.params.invoice_status));
```

Comment:
a last thing, please update `this.params.invoice_status` to `this.params.speakers_status`